src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -71,7 +74,7 @@ private IEnumerable<CodeRefactoringProvider> GetProviders(Document document)
             TextSpan state,
             CancellationToken cancellationToken)
         {
-            var extensionManager = document.Project.Solution.Workspace.Services.GetService<IExtensionManager>();
+            var extensionManager = document.Project.Solution.Workspace.Services.GetRequiredService<IExtensionManager>();
```

Comment:
Given you saw IExtensionManager was being null-checked earlier, should this be allowing a null return?

Reply:
The callsite that was passing null was not invoking GetService, but explicitly passing null as a named argument. I agree with @sharwell that existing code is wonky and should be refactored to avoid the null check completely, but it was just leading to bit too much refactoring to be included in this PR. I will capture this in the cleanup issue that I file.

Reply:
https://github.com/dotnet/roslyn/issues/40166